### ISSUE TITLE
Question about where the `user_id` is coming from in the documentation

### ISSUE THREAD

**Issue by yasuf** [RIPAGlobal/scimitar#33]
I'm looking at the docs here:
<img width="802" alt="Screen Shot 2022-12-07 at 2 08 58 PM" src="https://user-images.githubusercontent.com/7065942/206284997-cf6a6ca5-c4e6-4f1a-b405-1baa33dc074c.png">

`user_id` should be `record.id` right in that snippet?

**Comment by pond** [comment]
Correct, thanks. It's a copy and paste error from the `show` example above and is replicated in both `create` and `replace`, so both of those would need updating as suggested. In our own code base we're using `Scimitar::ActiveRecordBackedResourcesController` which has the "right thing" internally.
